src/SelfService.Tests/TestDoubles/StubCapabilityRepository.cs

```diff
@@ -35,4 +35,9 @@ public Task<IEnumerable<Capability>> GetAll()
     {
         throw new NotImplementedException();
     }
+
+    public Task<IEnumerable<Capability>> GetAllPendingDeletionFor(int days)
+    {
+        throw new NotImplementedException();
+    }
```

Comment:
Is this used in the test or does it need to be implemented because of an interface?